subsys/dm/Kconfig

```diff
@@ -0,0 +1,122 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if NRF_DM
+
+menu "nRF DM (Distance Measurement)"
+
+choice NRF_DM_TIMER
+	default NRF_DM_TIMER2
+	prompt "Timer to use for the nrf_dm library."
+
+config NRF_DM_TIMER0
+	depends on HAS_HW_NRF_TIMER0
+	bool "TIMER0"
+
+config NRF_DM_TIMER1
+	depends on HAS_HW_NRF_TIMER1
+	bool "TIMER1"
+
+config NRF_DM_TIMER2
+	depends on HAS_HW_NRF_TIMER2
+	bool "TIMER2"
+
+config NRF_DM_TIMER3
+	depends on HAS_HW_NRF_TIMER3
+	bool "TIMER3"
+
+config NRF_DM_TIMER4
+	depends on HAS_HW_NRF_TIMER4
+	bool "TIMER4"
+
+endchoice
+
+config DM_TIMESLOT_RESCHEDULE
+	bool "Timeslot reschedule"
+	default n
+	help
+	  "Enable allocation of the next timeslot after the previous timeslot has succeeded."
+
+config DM_INITIATOR_DELAY_US
+	int "Initiator start delay"
+	default 4000
+	help
+	  "Additional Initiator Start delay time. Used to improve synchronization."
+
+config DM_REFLECTOR_DELAY_US
+	int "Reflector start delay"
+	default 0
+	help
+	  "Additional Reflector Start delay time. Used to improve synchronization."
+
+config DM_INITIATOR_RANGING_WINDOW_US
+	int "Initiator ranging window"
+	default 19000
+	help
+	  "Time for ranging required by initiator role."
+
+config DM_REFLECTOR_RANGING_WINDOW_US
+	int "Reflector ranging window"
+	default 23000
+	help
+	  "Time for ranging required by reflector role."
+
+config DM_MIN_TIME_BETWEEN_TIMESLOTS_US
+	int "Minimum time between two timeslots"
+	default 8000
+	help
+	  "Minimum time between two timeslots.
+	  This should account for processing of the ranging data after the timeslot."
+
+config DM_RANGING_OFFSET_US
+	int "Ranging offset"
+	default 1200000
+	help
+	  "Time between the synchronisation and the ranging.
+	  Increasing this allows for more rangings to different peers but also increases latency."
+
+config DM_GPIO_DEBUG
+	bool "Debugging with output pins"
+	default n
+	select PPI_TRACE
+	help
+	  "Indication of distance measurement events by output pins."
+
+config DM_TIMESLOT_QUEUE_LENGTH
+	int "Timeslot queue length"
+	default 40
```

Comment:
Should we document somewhere what the influence of these (DM_TIMESLOT_QUEUE_LENGTH, DM_TIMESLOT_QUEUE_COUNT_SAME_PEER, DM_RANGING_OFFSET_US, DM_MIN_TIME_BETWEEN_TIMESLOTS_US, DM_REFLECTOR_RANGING_WINDOW_US, DM_INITIATOR_DELAY_US, ...) is? 
Like: how could you tune your setup to scale to many nodes? How can you reduce latency? etc.
And maybe we should also talk about restrictions on the values.

Reply:
I have added a parameters description in the documentation file.